secantTests/ZatoshiStringRepresentation/ZatoshiStringRepresentationTests.swift

```diff
@@ -28,63 +28,63 @@ final class ZatoshiStringRepresentationTests: XCTestCase {
         
         XCTAssertEqual(zatoshiStringRepresentation.mostSignificantDigits, "0.000")
         XCTAssertEqual(zatoshiStringRepresentation.leastSignificantDigits, "")
-        XCTAssertEqual(zatoshiStringRepresentation.feeFormat, "< 0.001")
+        XCTAssertEqual(zatoshiStringRepresentation.feeFormat, "(Typical Fee < 0.001)")
```

Comment:
Is there a way to obtain the text from string resources so we don't need to change all its occurrences in tests once the string resource is changed?

Reply:
Addressed, there's only 1 instance from now on, once we will support different languages, this Sting will be localized.